languages/common-lisp/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,122 @@
+## Comments
```

Comment:
I like the contents of this file! My only concern is that it is slightly big (compare it for example to the [C# basics](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/basics/.docs/introduction.md) one). This does not mean that it is wrong of course, but maybe some contents of this file could instead be moved to the `after.md` only? Another way to put it, is that the `introduction.md` document should provide the student with the minimal information needed to complete the exercise. The `after.md` document can then go into more detail if needed.

Reply:
@TheLostLambda Did you look at possibilities to perhaps shorten this introduction a bit? I could help with that if you'd like.

Reply:
I did move a couple of bits out of that file, but I'd love to hear if you have more suggestions of what I could shorten! It's been a bit of a challenge as I've been aiming for it to make sense on its own, but perhaps that is the wrong approach.